magprime/templates/static_views/cowbell.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+<head>
+    <title> Cowbell </title>
+</head>
+<body>
+
+<h3 align="center"> Cowbell </h3>
+
+<p> You'd be doing yourself a disservice, and every attendee of MAGFest, if you didn't buy the HELL out of this MAGFest branded cowbell.</p>
```

Comment:
Language, Nick, language!

Seriously though, will anyone be annoyed if they started filling out the page with their 12-year-old and they saw swearing? I'm not sure! I don't know. Just thought I'd point it out.